programs/launcher/main.cpp

```diff
@@ -353,8 +354,11 @@ struct launcher_def {
   void define_network ();
   void bind_nodes ();
   host_def *find_host (const string &name);
-  host_def *deploy_config_file (tn_node_def &node);
+  host_def *deploy_config_files (tn_node_def &node);
+  string compose_scp_command (const host_def &host, const bf::path &source,
+                              const bf::path &destination);
```

Comment:
At some point need to clean up these files so they follow the style of the rest of the code base. e.g. `const host_def& host, const bf::path& source, const bf::path& destination`